types/node/index.d.ts

```diff
@@ -5821,3 +5821,735 @@ declare module "async_hooks" {
      */
     export function createHook(options: HookCallbacks): AsyncHook;
 }
+
+declare module "http2" {
+    import * as events from "events";
+    import * as fs from "fs";
+    import * as net from "net";
+    import * as stream from "stream";
+    import * as tls from "tls";
+    import * as url from "url";
+
+    export interface Headers {
+        [headerField: string]: string | string[];
+    }
```

Comment:
I think it makes sense `http2` import some static method from `http` (`Header` interface in this case).

It's just like node `http2` core still import a few utility function from `http` (e.g, `httpConnectionListener` https://github.com/nodejs/node/blob/master/lib/internal/http2/core.js#L23)

@kjin What do you think? **import interface from http** or **duplicate header interface code**?